proc_echo_mw.py

```diff
@@ -47,7 +47,7 @@
     logger.info(strm("THIS IS THE SHAPE"))
     logger.info(strm(ndshape(s)))
     s = slice_FID_from_echo(s)['t2':(None,0.05)]
-    #fl.side_by_side('time domain (after filtering and phasing)\n$\\rightarrow$ use to adjust time range', s, time_range)
+    fl.side_by_side('time domain (after filtering and phasing)\n$\\rightarrow$ use to adjust time range', s, time_range)
     #}}}
```

Comment:
not clear why this was commented out

Reply:
forgot to uncomment prior to your pyspecdata fix, it was giving issues, but now it runs fine with it uncommented